internal/test_cases/tokenize_output_test_case.go

```diff
@@ -30,7 +30,10 @@ func (t *TokenizeTestCase) Run(executable *interpreter_executable.InterpreterExe
 	defer os.Remove(tmpFileName)
 
 	logger.Infof("Writing contents to ./test.lox:")
-	logger.UpdateSecondaryPrefix("test.lox")
+	existingSecondaryPrefix := logger.GetSecondaryPrefix()
+	// existingSecondaryPrefix can be "" or hold our intended log prefix
+	logger.UpdateSecondaryPrefix(existingSecondaryPrefix + "[test.lox] ")
```

Comment:
@ryan-gang this is hacking around the way secondary prefix works - I'd recommend keeping the interface as it was, and just changing the way we log here (append `[test.lox] ` to the log manually). 

The proper way to do this would be to allow a "stack" of secondary prefixes - and allow popping/pushing onto the stack. That seems like over-engineering at this point, wouldn't worry about designing for this until we have more such use-cases

Reply:
More reasoning on why this is a bad interface - `prefix` and `secondaryPrefix` are now treated differently. One is a plain string that will automatically be surrounded with braces and a leading space, the other is expected to include braces and a leading space within the value. This is not obvious from the interface, and one needs to look at the implementation to tell that they're used differently

Reply:
What should be the color of the appended `[test.lox] ` prefix then ?
If we want it to be yellow, should I copy over the `yellowColorize` function from `tester-utils`? Or expose it as public from there?

Reply:
Comparison of default and yellow colors. 
![image](https://github.com/codecrafters-io/interpreter-tester/assets/63599666/d754d63f-5026-42b2-bb27-e679cb982559)

Reply:
Ideally yellow, but it's okay if it's easier to do blue